engine/resources/cron.go

```diff
@@ -208,13 +220,22 @@ func (obj *CronRes) Close() error {
 
 // Watch for state changes and sends a message to the bus if there is a change.
 func (obj *CronRes) Watch() error {
+	var bus *dbus.Conn
+	var err error
+
 	// this resource depends on systemd
 	if !systemdUtil.IsRunningSystemd() {
 		return fmt.Errorf("systemd is not running")
 	}
 
 	// create a private message bus
-	bus, err := util.SystemBusPrivateUsable()
+	if obj.Session {
+		bus, err = util.SystemBusPrivateUsable()
+	} else {
+		bus, err = dbus.SessionBusPrivate()
+	}
+
+	fmt.Printf("!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!1111111111111111")
```

Comment:
!!!! Please remember to look at your diffs before submitting =D
My personal trick is that whenever I add debugging stuff that I don't want to commit I use XXX and then when I look at the diffs, I search for XXX and go through all of those. You could do that or use !!! or whatever you like!